worker/meson.build

```diff
@@ -173,6 +180,45 @@ common_sources = [
   'src/RTC/RTCP/XR.cpp',
   'src/RTC/RTCP/XrDelaySinceLastRr.cpp',
   'src/RTC/RTCP/XrReceiverReferenceTime.cpp',
+  'src/RTC/SCTP/Packet.cpp',
```

Comment:
What about avoiding compiling if the flag is not set for the time being?

Reply:
The flag is to enable or disable the custom SCTP stack in runtime. But I want to test and fuzz it despite it's not enabled in runtime. There is zero benefit in hiding those SCTP files and I want to keep working on SCTP and run tests and fuzzer in CI despite the SCTP stack is not enabled/used in production/runtime.

Reply:
I mean, yes, I can check the `ms_sctp_stack` option in `meson_options.txt` and then exclude SCTP files from `meson.build` and from `tests.cpp` in worker tests and from `fuzzer.cpp` in worker fuzzer, and then next week when I write another SCTP related PR I would need to edit CI yaml files to add that meson build option so SCTP is tested and fuzzed, etc etc. And all that complexity just to save ~8 seconds of build time.